src/argilla/client/feedback/dataset/remote/mixins.py

```diff
@@ -68,8 +68,7 @@ def __getitem__(
                 offsets = list(range(start, stop, FETCHING_BATCH_SIZE))
                 limits = [FETCHING_BATCH_SIZE] * len(offsets)
                 if stop % FETCHING_BATCH_SIZE != 0:
-                    offsets[-1] = stop - (stop % FETCHING_BATCH_SIZE) + 1
-                    limits[-1] = (stop % FETCHING_BATCH_SIZE) - 1
+                    limits[-1] = (stop % FETCHING_BATCH_SIZE)
```

Comment:
@alvarobartt  I think I'm not missing any other case. We should just adapt the last chunk limit starting at 1 instead of 0. So `%` result is enough.